Server/src/database.ts

```diff
@@ -1,41 +1,41 @@
 import {createConnection} from 'typeorm';
```

Comment:
I say we keep this file as is. The changes really don't need to be in the commit.

Reply:
i mean i just commented it out, the code is still there + in my DatasetEntity implemented